packages/components/inputs/number-input/src/number-input.tsx

```diff
@@ -151,13 +149,13 @@ NumberInput.toFormValue = (numberOrString) => {
   return '';
 };
 
-NumberInput.isEmpty = (value) => {
+NumberInput.isEmpty = (value: number | string): boolean => {
   if (typeof value === 'string') return value.trim().length === 0;
   if (typeof value === 'number') return isNaN(value);
   return true;
 };
 
-NumberInput.hasFractionDigits = (number) => {
+NumberInput.hasFractionDigits = (number: number) => {
```

Comment:
warning message sounds like i may be able to pass in a string type to this function, but the function signature only specifies numbers.

Reply:
that's my mistake. good catch!

Reply:
that's my mistake, and good catch on the message.